### PR TITLE
Choose library to sync from

### PR DESCRIPTION
**Pull request by mihneaz**
Added another command that enables user to select the library to sync from. 
I also reorganised the commands.

### CODE REVIEW

File: library-styles-sync.sketchplugin/Contents/Sketch/manifest.json [hunk 1]
```diff
@@ -10,10 +10,16 @@
...
+    "handler": "syncStylesWith",
+    "shortcut": "ctrl shift command j",
+    "name": "Sync With...",
+    "identifier": "com.zeroheight.library-styles-sync.sync-with"
+  },{
     "script": "script.js",
     "handler": "syncStyles",
     "shortcut": "ctrl command j",
-    "name": "Sync Library Styles",
+    "name": "Quick sync",
```
Comment (robintindale) [inline library-styles-sync.sketchplugin/Contents/Sketch/manifest.json]: how about `Sync with imported libraries`

File: library-styles-sync.sketchplugin/Contents/Sketch/manifest.json [hunk 2]
```diff
@@ -33,5 +39,22 @@
...
+  "menu": {
+    "items": [
+        {
+           "title": "Sync",
+           "items": [
+                "com.zeroheight.library-styles-sync.sync",
+                "com.zeroheight.library-styles-sync.sync-with",
+            ]
+        },{
+          "title": "JSON Sync",
```
Comment (robintindale) [inline library-styles-sync.sketchplugin/Contents/Sketch/manifest.json]: let's append `(experimental)`

File: library-styles-sync.sketchplugin/Contents/Sketch/script.js [hunk 3]
```diff
@@ -1,3 +1,42 @@
...
+  select.i18nObjectValues = options;
+  select.setEditable(false);
+  select.addItemsWithObjectValues(options);
+  select.selectItemAtIndex(0);
+  alert.addAccessoryView(select);
+
+  alert.addButtonWithTitle('Sync');
+	alert.addButtonWithTitle('Cancel');
+
+  if(alert.runModal() == "1000")
```
Comment (robintindale) [inline library-styles-sync.sketchplugin/Contents/Sketch/script.js]: does `NSAlertFirstButtonReturn` work? it's more semantic

**Comment by robintindale** [comment]
Great work :) do you also want to update the `readme`?

**Comment by mihneaz** [comment]
Thanks! :) I just added info about the new command to readme.

**Review by robintindale — APPROVED** [review]

**Pull request merged**